src/modules/edc-demo/components/empty-state/empty-state.component.html

```diff
@@ -0,0 +1,3 @@
+<div class="class=mt-[50px] w-full">
```

Comment:
`class="class=mt-`, the margin won't work right now, so I'd guess it's not intended here.